src/mongocxx/test/spec/util.cpp

```diff
@@ -130,12 +130,12 @@ void configure_fail_point(const client& client, document::view test) {
     }
 }
 
-void disable_fail_point(const client& client, stdx::string_view failpoint) {
+void disable_fail_point(const client& client, stdx::string_view fail_point) {
```

Comment:
Kept this consistent with `disable_fail_point`. Also, my idea raises warnings for misspellings.